db/schemaGenerator/data_generators.js

```diff
@@ -0,0 +1,88 @@
+const words = require('./words');
```

Comment:
Is this a library? I don't see it in the files or the JSON dependencies.

Reply:
Ohhhhh interesting. It's just a string with the nike ipsum words.
But I must have missed it when doing adding to git.